openshift-knative-operator/pkg/monitoring/rbac_proxy_injection_test.go

```diff
@@ -64,6 +66,24 @@ func TestInjectRbacProxyContainerToDeployments(t *testing.T) {
 		t.Fatalf("Unable to transform test manifest: %s", err)
 	}
 
+	limits := corev1.ResourceList{
+		corev1.ResourceMemory: resource.MustParse("100Mi"),
+		corev1.ResourceCPU:    resource.MustParse("100m"),
+	}
+	overrides := []base.WorkloadOverride{{
+		Name: "activator",
+		Resources: []base.ResourceRequirementsOverride{{
+			Container: RBACContainerName,
+			ResourceRequirements: corev1.ResourceRequirements{
+				Limits: limits,
+			},
+		}},
+	}}
+
+	if manifest, err = manifest.Transform(operator.OverridesTransform(overrides, nil)); err != nil {
```

Comment:
```suggestion
	if manifest, err = manifest.Transform(operator.OverridesTransform(KubeRbacProxyOverridesOnly(overrides, sets.NewString("activator")), nil)); err != nil {
```

I think `KubeRbacProxyOverridesOnly` should be covered, thought current test (upstream's `OverridesTransform()` only) ends up passing.

Reply:
```	overrides := []base.WorkloadOverride{{
		Name: "activator",
		Resources: []base.ResourceRequirementsOverride{{
			Container: RBACContainerName,
			ResourceRequirements: corev1.ResourceRequirements{
				Limits: limits,
			},
		}},
	}}
```
`KubeRbacProxyOverridesOnly` does just filtering based on deployment name which is supposed to have a kube-rbac-proxy. In this cases it makes no difference if we apply the filtering or not as the returned array is the same.